set_configmap.ps1

```diff
@@ -0,0 +1,19 @@
+$text = [string]::Format(((Get-Content $env:ACTION_PATH/octopus_configmap_old.yaml) -join "`n"),
+"$SPACE_NAME",
+"$PROJECT_NAME",
+"${env:GITHUB_REPOSITORY}",
+"$env:GITHUB_ACTOR",
+"${env:VERSION}",
+"${env:IMAGE_NAME}"
+)
+Set-Content -Path $env:ACTION_PATH/octopus_configmap.yaml -Value $text
+
+$deployScriptsPath = [System.IO.Path]::GetFullPath((Join-Path ${env:GITHUB_WORKSPACE} ${env:DEPLOY_SCRIPTS_PATH}))
+$chartsScriptsPath = [System.IO.Path]::GetFullPath((Join-Path $deployScriptsPath ${env:CHARTS_DIR_PATH}))
+$repoName =  basename $(git remote get-url origin) .git
```

Comment:
What is the purpose of this?

Reply:
Separated to each line so that it will be clear. Anyways this is not needed.